### DISCUSSION TITLE
Use System Date Format (with 24hr time)

### DISCUSSION THREAD

**Discussion by ketonik** [spacedriveapp/spacedrive#2389] [Feature Request]
Can we please have the date format respect the actual system it's running on, that'd mean everywhere outside those 51 states.
Here, it's DD/MM/YYYY, and since Windows is so dumb it hasn't got a simple switch to enable 24hr time format anywhere in Settings,  please add this to Spacedrive. thx

I kinda know what month it is from the 1st day of the month, and i'm gona know that for the remaining 30 days of the month, but i'm asking every day what's todays date, so don't need to say or see the month every day. meh...US logic...IDK

**Comment by utkubakir** [comment]
What does it show for you now? Some screenshots and example of the expected format would be nice.

**Comment by ketonik** [comment]
Some screenies, sure:
Spacedrive List View showing non system date format.png

![Spacedrive List View showing non system date format](https://github.com/spacedriveapp/spacedrive/assets/49221849/002656d2-78b5-4880-8b84-990020a1a8a0)

macOS Finder date in List View, it's smarter than Windows, the Date Columns will display short medium or long dates and those contract and expand based on the width pf the columns, examples below:
![1 macOS Finder Date List View wide](https://github.com/spacedriveapp/spacedrive/assets/49221849/1e74278d-be5c-4d07-85d0-70fdd80d71ee)
![2 macOS Finder Date List View Med](https://github.com/spacedriveapp/spacedrive/assets/49221849/715884c5-8a8a-41ff-b748-b44003d4e3e7)
![3 macOS Finder Date List Viiew narrow](https://github.com/spacedriveapp/spacedrive/assets/49221849/a7955593-0750-4fef-88af-ea44a255e077)


Here is File Explorer:
![image](https://github.com/spacedriveapp/spacedrive/assets/49221849/52409a6b-fcbc-4c37-bf43-f53c9d56390e)

**Comment by utkubakir** [comment]
https://github.com/spacedriveapp/spacedrive/issues/2406 Will be solved here

**Comment by artsiom-voitas** [comment]
@ketonik Date format feature will be available in the next release!

**Comment by ketonik** [comment]
Thx dev’s wikid, sortid the date out propa in 0.2.14On 1 May 2024, at 02:59, Artsiom Voitas ***@***.***> wrote:﻿
@ketonik Date format feature will be available in the next release!
